### PR TITLE
tau->Pl implementation

### PR DESCRIPTION
**Pull request by jackypheno**
Adding branching fractions of the following decays:

tau-> pie
tau->pimu 
tau->Ke 
tau->Kmu

### CODE REVIEW

**Comment by jackypheno** [comment]
@DavidMStraub OK, this is the new pull request which replaces the previous one for the same purpose.
Now, I have added the full calculation of tau->Pl including scalar and vector contributions. Also a unittest is added. Once the cross checks are done you can merge it.

**Comment by jackypheno** [comment]
Oh, I see all the checks being failed. Perhpas its due to the fact that I have not yet added `from . import taupl` to  the file `flavio/physics/taudecays/__init__.py`  ?

**Comment by DavidMStraub** [comment]
Thanks for the PR!
Yes, for SM prediction to work you need to add it to the init.

File: flavio/physics/taudecays/taupl.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+       C9m =  -wc['C9_'+qqll]  + wc['C9p_'+qqll]
+       C10m = -wc['C10_'+qqll] + wc['C10p_'+qqll]
+       CSm =   wc['CS_'+qqll]  - wc['CSp_'+qqll]
+       CPm =   wc['CP_'+qqll]  - wc['CPp_'+qqll]
+       Kv = xi_t * 4*GF/sqrt(2) * alphaem/(4*pi)
+       Ks = Kv * mq2
+       gV = - fP* Kv * C9m/2
+       gA = - fP* Kv * C10m/2
+       gS =   fP* Ks * mP**2 * CSm/2 / (mq1+mq2)
+       gP =   fP* Ks * mP**2 * CPm/2 / (mq1-mq2)
```
Comment (DavidMStraub) [inline flavio/physics/taudecays/taupl.py]: @jackypheno should this be a + in the denominator as well?

Comment (jackypheno) [reply flavio/physics/taudecays/taupl.py]: @DavidMStraub  Oh, yes. Thanks.

**Comment by jackypheno** [comment]
@DavidMStraub I think you can merge it now.

**Comment by DavidMStraub** [comment]
Thanks! :tada:

**Pull request merged**